### DISCUSSION TITLE
[react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!

### DISCUSSION THREAD

**Discussion by erez-unitronics** [software-mansion/react-native-gesture-handler#1830] [Q&A]
Hello,

I've been upgraded the library to the new version: 2.2.0 And now I've got this warning:
"[react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!"

Should I do something?
Thanks!

**Comment by Pietro-Putelli** [comment]
This warning happens because you're using the new version of `react-native-gesture-handler`, but in your code there are some components that use the old one.
For instance if your code contains `PanGestureHandler` from `rngh1` you should update to `Gesture.Pan()`.
https://docs.swmansion.com/react-native-gesture-handler/docs/quickstart/quickstart

If you don't update it simply mute the warning using:
```
LogBox.ignoreLogs([
  "[react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!",
]);
```

I hope it was helpful to you
